frontend/pages/admin/OrgSettingsPage/cards/Advanced/Advanced.tsx

```diff
@@ -298,7 +299,7 @@ const Advanced = ({
             Disable scripts
           </Checkbox>
           <Checkbox
-            onChange={handleInputChange}
+            onChange={onInputChange}
```

Comment:
For context: `main` had a refactor name change from `handleInputChange` to `onInputChange`, that's all.